### PR TITLE
added delete confirmation for url shortener

### PR DESCRIPTION
**Pull request by syng828**
Used code from UserManager (Overview.js) to add in delete confirmation for URL shortener

### CODE REVIEW

File: src/Pages/URLShortener/URLShortener.js [hunk 1]
```diff
@@ -344,81 +348,100 @@ export default function URLShortenerPage(props) {
...
+        cancelText: 'No, it\'s chill',
+        confirmClassAddons: 'bg-red-600 hover:bg-red-500',
+        handleConfirmation: () => {
+          handleDeleteUrl(urlToDelete.alias);
+          setToggleDelete(!toggleDelete);
+        },
+        open: toggleDelete
+      }
+      } />
+      <div className='overview-container bg-gradient-to-r from-gray-800 to-gray-600 min-h-[100dvh]'>
```
Comment (banyar-shin) [inline src/Pages/URLShortener/URLShortener.js:365]: No need to make a div here, overview container-div is already made on line 350.

**Review by banyar-shin — CHANGES_REQUESTED** [review]
Make changes according to comments.

**Review by banyar-shin — APPROVED** [review]
looks good to me!

File: src/Pages/URLShortener/URLShortener.js [hunk 2]
```diff
@@ -344,6 +348,20 @@ export default function URLShortenerPage(props) {
...
   // https://tailwindui.com/components/application-ui/forms/form-layouts
     <div className='overview-container bg-gradient-to-r from-gray-800 to-gray-600 min-h-[100dvh]'>
+      <ConfirmationModal {... {
+        headerText: `Delete ${urlToDelete.alias} for ${urlToDelete.url} ?`,
+        bodyText: `Are you sure you want to delete 
+          ${urlToDelete.alias}? It'll be gone forever if you do.`,
+        confirmText: `Yes, goodbye ${urlToDelete.alias}`,
+        cancelText: 'No, it\'s chill',
+        confirmClassAddons: 'bg-red-600 hover:bg-red-500',
+        handleConfirmation: () => {
```
Comment (evanugarte) [inline src/Pages/URLShortener/URLShortener.js:358]: what about when they cancel? does the modal close?

File: src/Pages/URLShortener/URLShortener.js [hunk 3]
```diff
@@ -344,6 +348,20 @@ export default function URLShortenerPage(props) {
   // the below input layout is taken from
   // https://tailwindui.com/components/application-ui/forms/form-layouts
     <div className='overview-container bg-gradient-to-r from-gray-800 to-gray-600 min-h-[100dvh]'>
+      <ConfirmationModal {... {
+        headerText: `Delete ${urlToDelete.alias} for ${urlToDelete.url} ?`,
+        bodyText: `Are you sure you want to delete 
+          ${urlToDelete.alias}? It'll be gone forever if you do.`,
+        confirmText: `Yes, goodbye ${urlToDelete.alias}`,
+        cancelText: 'No, it\'s chill',
```
Comment (evanugarte) [inline src/Pages/URLShortener/URLShortener.js:356]: `cancelText` defaults to `Cancel`, so we can omit it as a prop
```suggestion
        confirmText: `Yes, delete ${urlToDelete.alias}`,
```

**Review by pnavab — APPROVED** [review]
looks good now

File: src/Pages/URLShortener/URLShortener.js (hunk 2 above)
Comment (syng828) [reply src/Pages/URLShortener/URLShortener.js:357]: done

**Pull request merged**